features/myquicktest.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Test features
+  # @author kewang@redhat.com
```

Comment:
@wangke19 this looks like a `destructive` action?

Reply:
@pruan-rht, please ignore my request, my local branch is a little messy.

Reply:
I will close it.